GenAI Builder/projects/genAITutorial.json

```diff
@@ -0,0 +1,389 @@
+{
```

Comment:
should add a project description with details about the use of this project/tutorial

may need to pull this from the eventual docs-page for the tutorial

Reply:
Will add, but it will basically just be a repeat of the readme, with a reference to the tutorial.  The service has no "purpose" and isn't a functioning application, just a collection of examples.